clients/swift/Tests/MyanmarToolsTests/MyanmarToolsTests.swift

```diff
@@ -44,7 +44,6 @@ final class MyanmarToolsTests: XCTestCase {
           }
           let expectedProbability = try XCTUnwrap(Double(elements[0]))
           let input = elements[1]
-          print("Checking getZawgyiProbability(\(input)) = \(expectedProbability)")
```

Comment:
I actually like having this line because it makes me confident looking at the Travis logs that there is actually something being tested in this file. If you delete it, can you add maybe a summary line that says "X tests passed" or something like that?

Reply:
If it's useful, I'm happy to leave it. Restored back to the way it was.